lib/KTable/index.vue

```diff
@@ -710,7 +710,7 @@
 
   .k-table {
     width: 100%;
-    border-collapse: collapse;
+    table-layout: fixed;
```

Comment:
Any specific reason for removing the border-collapse feature?
Also, fixing the table layout disables the scroll behaviour -  and that is a feature we need. So, can you please revert the table-layout fixed property. Thanks!